null_safety_examples/misc/test/library_tour/core_test.dart

```diff
@@ -233,10 +233,10 @@ void main() {
       fruits.clear();
       assert(fruits.isEmpty);
 
-      // You can also create a List using one of the constructors.
+      // You can also create a List using one of the factory constructors.
```

Comment:
I'm not 100% sure if it's needed to qualify the constructors as factory. It's [technically correct](https://i.kym-cdn.com/photos/images/original/000/909/991/48c.jpg), of course, but may imply the existence of non-factory constructors.

Here, we're talking about constructors, specifically, because, above, we've been using literals.

```suggestion
      // You can also create a List using one of the constructors.
```